platform/android/export/export_plugin.cpp

```diff
@@ -2072,8 +2073,10 @@ Error EditorExportPlatformAndroid::run(const Ref<EditorExportPreset> &p_preset,
 				print_line("Reverse result2: " + itos(rv));
 			}
 		} else {
-			static const char *const msg = "--- Device API < 21; debugging over Wi-Fi ---";
```

Comment:
You deleted this line but didn't fix any places using it

Reply:
Fixed!